src/components/img.ts

```diff
@@ -18,14 +19,19 @@ interface Props {
     format?: Format;
 }
 
-const styles = (format?: Format): SerializedStyles => css`
-    background-color: ${format?.design === Design.Media ? neutral[20] : neutral[97]};
-
-    ${darkModeCss`
+const styles = (role: Option<Role>, format?: Format): SerializedStyles => {
+    const darkModeStyles = darkModeCss`
         color: ${neutral[60]};
         background-color: ${neutral[20]};
-    `}
-`;
+    `
+
+    return role.fmap(imageRole => imageRole).withDefault(Role.HalfWidth) === Role.Thumbnail
+        ? darkModeStyles
+        : css`
+            background-color: ${format?.design === Design.Media ? neutral[20] : neutral[97]};
+            ${darkModeStyles}
+        `
+}
```

Comment:
I'm finding it a little bit difficult to follow the logic here. Is it worth splitting it into a couple of functions? Something like:

```ts
const backgroundColour = (role: Role, format: Format): Option<SerializedStyles> => {
    if (imageRole === Role.Thumbnail) {
        const colour = format.design === Design.Media ? neutral[20] : neutral[97];
        return new Some(css`background-color: ${colour};`);
    }

    return new None();
}

const darkStyles = darkModeCss`
    color: ${neutral[60]};
    background-color: ${neutral[20]};
`;

const styles = (role: Option<Role>, format: Format): SerializedStyles => css`
    ${role.andThen(backgroundColour).withDefault('')}
    ${darkStyles}
`;
```